chat_shell/tests/test_kb_prompt_deduplication.py

```
@@ -223,6 +223,50 @@ async def test_kb_prompt_does_not_contain_placeholder(self):
+            mock_kb_tool_class.return_value = MagicMock()
+
+            result = await prepare_knowledge_base_tools(
+                knowledge_base_ids=[1],
+                user_id=1,
+                db=MagicMock(),
+                base_system_prompt="Base",
+                model_id="claude-3-5-sonnet",
+                model_config={"model_id": "claude-3-5-sonnet", "api_key": "k"},
+                skip_prompt_enhancement=False,
+                is_user_selected=True,
+                auth_token="jwt-token",
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to argument: "auth_token"

(S106)